### PR TITLE
website: intro punchline

### PR DESCRIPTION
**Pull request by themr0c**
### What does this PR do?

Website intro punchlines.

### Screenshot/screencast of this PR

Screenshot from 2023-05-22 14:15:28 Selection 783x334 01

### CODE REVIEW

File: website/src/pages/index.tsx [hunk 1]
```diff
@@ -102,10 +102,12 @@ function Hero() {
             <h1 className="title-font sm:text-4xl text-3xl lg:text-6xl mb-8 font-medium text-gray-900 dark:text-white">
               Containers and Kubernetes for application developers
             </h1>
-            <p className="text-base md:text-lg">
-              Podman Desktop is an open source graphical tool enabling you to seamlessly work with containers and
-              Kubernetes from your local environment.
-            </p>
+            <p className="text-base md:text-lg">Your simplistic onboarding to go from containers up to Kubernetes.</p>
```
Comment (deboer-tim) [inline website/src/pages/index.tsx:105]: 'Simplistic' has the connotation of being overly simplified or naive, would prefer not to use it.

File: website/src/pages/index.tsx [hunk 2]
```diff
@@ -102,10 +102,12 @@ function Hero() {
...
+            <p>Open source</p>
+            <p>Graphical user interface</p>
+            <p>On your developer environment</p>
```
Comment (deboer-tim) [inline website/src/pages/index.tsx:108]: Not clear what this means, did you mean 'developer machine'?

File: website/src/pages/index.tsx [hunk 3]
```diff
@@ -102,10 +102,12 @@ function Hero() {
...
+            <p>To work with containers and Kubernetes</p>
+            <p>Abstracting away container and Kubernetes tooling configuration</p>
```
Comment (deboer-tim) [inline website/src/pages/index.tsx:110]: Maybe just 'tool' in this context? Or remove it entirely?

**Review by deboer-tim — CHANGES_REQUESTED** [review]
TBH I prefer the original text. I'm not opposed to rewording, but I think we should keep it to a tagline vs multiple bullets.

**Comment by benoitf** [comment]
I don't like as well the 'bullet without bullets' style

**Comment by mairin** [comment]
(grain of salt: I am not an SEO person at all)

The current slug under the headline is "Podman Desktop is an open source graphical tool enabling you to seamlessly work with containers and Kubernetes from your local environment." I think this is a good approach because the headline is a bit more abstract and high level (containers to kubernetes), and the slug under says what it actually is ("an open source graphical tool")

The new suggestion has this but repeats the headline in different language first, then talks about it being an open source graphical tool. When I first saw it, it just seemed like an overwhelming amount of text - so I think to go with the suggested language changes might require a different layout.

@themr0c is there a specific goal or goals you're trying to target with the suggested changes on this? (for example, SEO :) ) it could help me figure out a potentially better layout. here is an example of how it could be laid out (i made some changes to the text tho so the layout would work and tried to account for @deboer-tim 's feedback)

(I worry about this one though because doesnt state clearly upfront what Podman Desktop is)
Screenshot from 2023-05-22 21-19-30

**Comment by benoitf** [comment]
closing due to lack of activity, could be reopened

**Pull request closed without merging**